SWIG_CGAL/Alpha_shape_2/CGAL_Alpha_shape_2.i

```diff
@@ -20,6 +20,7 @@ SWIG_CGAL_package_common()
 %import  "SWIG_CGAL/Triangulation_2/Reference_wrappers.i"
 %include "SWIG_CGAL/Common/Iterator.h"
 
+%ignore CGAL_VERSION
```

Comment:
Maybe `%ignore CGAL_GIT_HASH` as well?

Reply:
seems that the macros are parsed before being ignored